### DISCUSSION TITLE
가장 최근에 remote에 push한 커밋 삭제하기

### DISCUSSION THREAD

**Discussion by compuTasha** [DeveloperAcademy-POSTECH/MacC-GetARock#58] [개발 팁]
땡큐하게도 가장 최근 push한 커밋만 삭제하면 된다면.. 이지하게 해결할 수 있습니다...^^

## 🚀 삭제할 커밋 확인

`git log`

### git reset

`git reset HEAD^`
커밋 내용을 삭제하고, 삭제한 커밋은 이력에서 보이지 않게 된다.

### git revert

`git revert HEAD^`
커밋 내용을 삭제하고, 삭제한 커밋은 이력에서 보인다.
이미 push한 상태에서는 revert를 쓴다.

### git push-f origin BRANCH

`git push-f origin BRANCH`
커밋을 삭제했다는 것을 깃허브 서버에 전달하여 깃허브 내에서도 해당 커밋을 삭제하도록 한다.  

<br>  

[참고 링크](https://velog.io/@gygy/resetrevert-push한-commit-삭제하기)

for you.. @taehyeonk

**Comment by GODNOEL** [comment]
당신 뭐야 하늘에서 떨어진 천사인건가? 👼
